types/freshy/freshy-tests.ts

```diff
@@ -0,0 +1,27 @@
+import { unload, reload, freshy } from 'freshy';
+import minimist = require('minimist');
+
+declare function require(x: string): any;
+
+unload('minimist'); // $ExpectType boolean
+
+const reloaded = reload('minimist'); // $ExpectType any
+// $ExpectType boolean
+minimist === reloaded; // false
+
+const freshlyLoaded = freshy('minimist'); // $ExpectType any
+// $ExpectType boolean
+minimist === freshlyLoaded; // false
+
+let alsofresh: any;
+// $ExpectType any
+const fresh = freshy('minimist', (fresh) => {
+    alsofresh = require('minimist');
+
+    // $ExpectType boolean
+    fresh === alsofresh; // true
+});
+// $ExpectType boolean
```

Comment:
An `$ExpectType` on an `===` expression doesn't really test the library types, since `===` returns a boolean regardless of what its operands are.